datamodel/src/test/java/com/hartwig/serve/datamodel/DatamodelTestFactory.java

```diff
@@ -13,22 +13,69 @@ public final class DatamodelTestFactory {
     private DatamodelTestFactory() {
     }
 
+    @NotNull
+    public static ImmutableClinicalTrial.Builder clinicalTrialBuilderBuilder() {
+        return ImmutableClinicalTrial.builder()
+                .studyNctId("studyNctId")
+                .studyTitle("studyTitle")
+                .countriesOfStudy(Sets.newHashSet("countriesOfStudy"))
+                .therapyName("therapyName");
+    }
+
     @NotNull
     public static ImmutableTreatment.Builder treatmentBuilder() {
-        return ImmutableTreatment.builder().name(Strings.EMPTY);
+        return ImmutableTreatment.builder().name("name");
     }
 
     @NotNull
     public static ImmutableCancerType.Builder cancerTypeBuilder() {
         return ImmutableCancerType.builder().name(Strings.EMPTY).doid(Strings.EMPTY);
     }
 
+    @NotNull
+    public static Treatment treatmentBuilder(ActionableEvent event) {
+        Treatment treatment = null;
+        if (event.intervention() instanceof Treatment) {
+            treatment = (Treatment) event.intervention();
+        }
+
+        if (treatment == null) {
+            throw new IllegalStateException("An actionable event has to contain treatment");
+        }
+        return treatment;
+    }
+
+    @NotNull
+    public static ClinicalTrial clinicalTrialBuilder(ActionableEvent event) {
+        ClinicalTrial clinicalTrial = null;
+        if (event.intervention() instanceof ClinicalTrial) {
+            clinicalTrial = (ClinicalTrial) event.intervention();
+        }
+
+        if (clinicalTrial == null) {
+            throw new IllegalStateException("An actionable event has to contain clinical trial");
+        }
```

Comment:
I don't think this check is necessary since the object is not null

Reply:
I leave this check in because or treatment of clinical trial could be nullable